config/nginx/docker.conf

```diff
@@ -0,0 +1,21 @@
+server {
```

Comment:
This file is slightly different from the original nginx.conf, I'd prefer them be the same if possible